File: OP_labs_cs/lab2_cs/lab2_cs/ContentFunctions.cs
using System;
using System.Collections.Generic;

namespace lab2_cs
{
    public class ContentFunctions
    {
        public static void EnterAbits(string fileName)
        {
            string checkEntering;
            do
            {
                Console.WriteLine("\nEnter abit's fullname: ");
                string fullName = Console.ReadLine();
                Console.WriteLine("Enter abit's birthday: [dd.mm.yyyy]");
                string birthDay = Console.ReadLine();
                Console.WriteLine("Enter abit's sex: [m/f]");
                string Sex = Console.ReadLine();
                FileFunctions.WriteToFile(fileName, fullName, birthDay, Sex);
                Console.WriteLine("\nContinue entering?" +
                                  "\n[y/n]");
                checkEntering = Console.ReadLine();
            } while (checkEntering == "y" || checkEntering == "Y");
        }
        public static int[] SplitBirthday(string birthStr)
        {
            string[] splitedStr = birthStr.Split('.');
            int[] birthArr = {
                Convert.ToInt32(splitedStr[0]),
                Convert.ToInt32(splitedStr[1]),
                Convert.ToInt32(splitedStr[2])
            };
            return birthArr;
        }
        public static List<Abit> DeleteOldAbit(List<Abit> AbitsList)
        {
            List<Abit> youngAbits = new List<Abit>();
            foreach (var abit in AbitsList)
            {
                if (abit.Age() <= 35)
                {
                    youngAbits.Add(abit);
                }
            }
            return youngAbits;
        }
        public static void PrintList(List<Abit> AbitsList)
        {
            foreach (var abit in AbitsList)
            {
                Console.WriteLine(abit.Info());
            }
        }
        public static void CreateRecruitList(List<Abit> AbitsList, string fileName)
        {
            foreach (var abit in AbitsList)
            { 
                if (abit.Sex == Abit.SexType.Male && abit.Age() >= 18 && abit.Age() <= 27)
                {
                    FileFunctions.WriteToFile(fileName, abit.FullName, abit.BirthDay.ToString("d"), abit.Sex.ToString());
                }
            }
        }
    }
}

File: OP_labs_py/lab2_py/ContentFunctions.py
from FileFunctions import *


def entering_to_file(file_name: str):
    cont = True
    print("Enter abit's list: ")
    while cont:
        full_name = input("Fullname: ")
        birthday = input("Birthday [dd.mm.yyyy]: ")
        sex_letter = input("Sex: ")
        sex = "Male" if sex_letter == "m" or sex_letter == "M" else "Female"
        write_to_file(file_name, full_name, birthday, sex)
        check_entering = input("Continue entering? [y/n]\n")
        cont = True if check_entering == "y" or check_entering == "Y" else False


def delete_older_abits(abits_list: list):
    for abit in abits_list:
        if abit.get_age() > 35:
            abits_list.remove(abit)


def print_list(abits_list: list):
    for abit in abits_list:
        print(abit.get_info())


def recruits(abits_list: list, file_name: str):
    for abit in abits_list:
        if 18 <= abit.get_age() <= 27 and abit.sexType == "Male":
            write_to_file(file_name, abit.fullName, abit.birthDay, abit.sexType)
